jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -605,6 +606,50 @@ def vue_set_value(self, data):
         value = data.get('value')
         setattr(self, attr_name, value)
 
+    def vue_apply_RGB_presets(self, data):

```

Comment:
would this be useful from the public API (in which case it should be a separate method which `vue_...` calls and then added to the list of exposed methods for Imviz).

Reply:
Good call, that's definitely a good idea.

Reply:
Done